silhouette/app/com/mohiva/play/silhouette/api/services/AuthenticatorService.scala

```diff
@@ -128,7 +150,7 @@ trait AuthenticatorService[T <: Authenticator] {
    * @param authenticator The authenticator to touch.
    * @return The touched authenticator on the left or the untouched authenticator on the right.
    */
-  protected[silhouette] def touch(authenticator: T): Either[T, T]
+  def touch(authenticator: T): Either[T, T]
```

Comment:
Renew and touch are two different things. Renew discards the current authenitcator, then it creates a new one, initializes it and embeds it into the result. Touch updates only the `idleTimeout` so that the authenticator cannot time out. And if the `idleTimeout` was updated then we must update the authenticator client side. Therefore we must call the `update` method on the authenticator.